src/MPQEditor/MPQEditor.ts

```diff
@@ -716,4 +740,32 @@ export class MPQEditorProvider
       this.showCircleModelGraph(document, webviewPanel);
     }
   }
+
+  /**
+   * @brief Called when user requests to load visq file from UI
+   */
+  private handleLoadVisqFile(
+    document: vscode.TextDocument,
+    webviewPanel: vscode.WebviewPanel
+  ) {
+    const dialogOptions = {
+      canSelectMany: false,
+      canSelectFolders: false,
+      openLabel: "Open",
+      filters: { "target files": ["visq.json"], "all files": ["*"] },
```

Comment:
```suggestion
      filters: { "target files": ["visq.json"], "all files": ["*"]},
      title: "Open VISQ File" };
```

Reply:
Without this, the system shows

> "" is ready.

In the dialog.

Reply:
Ahhh. Sorry. Thank you very much!